scripts/ci/build_demo_app.py

```diff
@@ -218,7 +308,7 @@ def main() -> None:
 BASE_PATH = "web_demo"
 SCRIPT_PATH = os.path.dirname(os.path.relpath(__file__))
 # When adding examples, add their requirements to `requirements-web-demo.txt`
-EXAMPLES = {
+EXAMPLES: dict[str, Any] = {
```

Comment:
@abey79 
We didn't specify this, but I assume you wanted the examples which are on `demo.rerun.io`, because those are the only ones which have `.rrd` files built on CI. If that's the case, the `dna` example needs a README + thumbnail - how did we make those again?

Reply:
I'm thinking that it's maybe more future-proof to include everything in the manifest. Then, in the UI, we can pull only those example which have a RRD url specified.

As for the DNA example, I guess we just need to add a `README.md` with front matter in `examples/python/dna/`? It looks like an oversight that there isn't one already.

Reply:
> I'm thinking that it's maybe more future-proof to include everything in the manifest

That's not a difficult change to make (we fetch all the data from README files instead of hardcoding a map like this), so I'd rather only do the minimum required work here.

> As for the DNA example, I guess we just need to add a README.md with front matter in examples/python/dna/?

Yes, but that requires taking a screenshot which I haven't done before, do you remember where the instructions are for that?

Reply:
The script doesnt assume that the readme exists, so its safe to add it later.

Reply:
Taking a screenshot is `ctrl-P` for the command palette -> screenshot. For the upload, `just upload --help`. Since the screenshot feature copies to clipboard, the easiest is `just upload --name example_name` (the script pulls from clipboard if no input path is provided).